src/main/java/com/faforever/api/clan/ClanService.java

```diff
@@ -83,6 +83,12 @@ Clan create(String name, String tag, String description, Player creator) {
     clanRepository.save(clan);
     return clan;
   }
+  
+  @SneakyThrows
+  String generatePlayerInvitationToken(Player requester, int clanId) {
+    // Playerless invitation
+    return generatePlayerInvitationToken(requester, 0, int clanId);
```

Comment:
You can't have `int clanId`

Reply:
ah, oops
but i dont even know if overloading is the right thing to do here

Reply:
Why can u not? And yeah overloading is fine